src/middlewared/middlewared/plugins/smb.py

```diff
@@ -564,6 +570,7 @@ async def configure(self, job, create_paths=True):
         This step is not required when underlying database is clustered (cluster node should
         just recover with info from other nodes on reboot).
         """
+        self.configured = True
```

Comment:
You're going to want to set the class attribute here. (i.e. instead of `self.configured`, do `SMBService.configured`) so that all instantiations of this class get the same value for that attribute. You can see past tribulations that I encountered doing something similar in ticket NAS-109597. Also, when is this ever set back to `False`?